### DISCUSSION TITLE
Decorators.panel() is showing in bottom after update dotnet 6  to dotnet 8

### DISCUSSION THREAD

**Discussion by arunkumar12712** [serenity-is/Serenity#7097] [General]
<img width="1244" alt="image" src="https://github.com/serenity-is/Serenity/assets/15244940/b47af5c7-21a8-40b3-a578-fc324c8b5550">

**Comment by LSNicholls** [comment]
@arunkumar12712 , I had the same experience, which versions of the Serenity packages are you using?  I was okay with 8.1.2 but when I went up to 8.3.5 it started happening.  Also, what version of sergen is in dotnet-tools.json?

**Reply by arunkumar12712** [reply]
<img width="616" alt="image" src="https://github.com/serenity-is/Serenity/assets/15244940/bdd4c58c-2c4d-4a7e-8919-3b14995f05bf">

**Reply by LSNicholls** [reply]
@arunkumar12712 , Thanks for checking this. (Why does your Serenity.Net.Web item have an "x" next to it?)

Try updating the 8.3.7 ones to 8.3.8 - that fixed some related stuff for me.

And check your .config/dotnet-tools.json to find the version of sergen.  I have 8.3.5 and it seems to be working.

**Reply by arunkumar12712** [reply]
.config/dotnet-tools.json version 6.9.3 after update 8.3.5 and also update 8.3.7 to 8.3.8 then not working .
but i have download common-style.css version of 6.3.9 then replace "~/Serenity.Extensions/common-style.css" to "~/Content/site/common-style.css" now working.

**Reply by LSNicholls** [reply]
Thanks @arunkumar12712 .  I guess that updating some of the Serenity packages to 8.3.8 somehow fixed this for me, though.  Does your Serenity.Net.Web (mine is at 8.3.8) still have an "X" next to it like your screenshot?

**Comment by volkanceylan** [comment]
We don't recommend updating Serene to recent versions until we release a compatible version as too many things have changed, and Serene is still based on .NET 6 and it has jquery etc. dependencies.

**Comment by LSNicholls** [comment]
Thanks @volkanceylan.  I found that out the hard way.
